package.json

```
@@ -93,6 +112,16 @@
           "default": null,
           "description": "Set a custom path to the Zig binary. Defaults to 'zig' in your PATH."
         },
+        "zig.testArgs": {
+          "type": "string",
+          "default": "",
+          "description": "Additional flags to pass \"zig test\""
+        },
+        "zig.disableProblemMatcherForTest": {
+          "type": "bool",
```

```suggestion
          "type": "boolean",
```
This fixes the `Incorrect type. Expected "bool".` error when trying to use this setting.